src/Features/CSharp/Portable/DocumentationComments/CSharpDocumentationCommentSnippetService.cs

```diff
@@ -94,12 +95,20 @@ protected override int GetPrecedingDocumentationCommentCount(MemberDeclarationSy
             return count;
         }
 
-        protected override List<string> GetDocumentationCommentStubLines(MemberDeclarationSyntax member)
+        protected override List<string> GetDocumentationCommentStubLines(MemberDeclarationSyntax member, string existingCommentText)
         {
+            // Ensure there is at least one space after the ///
+            existingCommentText = existingCommentText switch
+            {
+                { Length: 0 } => " ",
+                [not ' ', ..] => " " + existingCommentText,
+                _ => existingCommentText
+            };
```

Comment:
fwiw, i think a conditional of `existingCommentText.StartsWith(' ') ? existingCommentText : " " + existingCommentText` is clearer.  the above took me a lot of time to figure out.